elmslie-samples/compose-paging/src/main/java/vivid/money/elmslie/samples/android/compose/view/PagingFragment.kt

```diff
@@ -29,16 +29,16 @@ class PagingFragment : ElmComponentFragment<PagingEvent, PagingEffect, PagingSta
     @Preview
     @Composable
     private fun Screen() {
-        val state by state()
-        val effect by effect()
-        PagingScreen(
-            state = state,
-            effect = effect,
-            onRefresh = { store.accept(Ui.PullToRefresh) },
-            onReloadPage = { store.accept(Ui.ClickReloadPage) },
-            onReloadScreen = { store.accept(Ui.ClickReloadScreen) },
-            onCloseToEnd = { store.accept(Ui.CloseToEnd) }
-        )
+        store.states()
+        store.effects()
+//        PagingScreen(
```

Comment:
commented code

Reply:
it's tempopaly, for this iteration it's ok